tools/replay/parse_dump_resources_cli.cpp

```diff
@@ -78,51 +78,42 @@ static bool CheckIndicesForErrors(const gfxrecon::decode::VulkanReplayOptions& v
     // Indices should be provided sorted
     if (vulkan_replay_options.Draw_Indices.size())
     {
-        bool is_2d = false;
         for (const auto& indices : vulkan_replay_options.Draw_Indices)
         {
-            is_2d |= (indices.size() > 0);
```

Comment:
DrawIndices must be 2d array

Reply:
Fix in 2d8a134f8a9f952fd31a128e78d2b0b455f1a261